src/pages/wiki/[...pageSlug]/index.tsx

```diff
@@ -34,6 +38,13 @@ const StyledGrid = styled.div`
     }
 `;
 
+const ArrowLink = styled(Link)`
+    align-self: flex-start;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    margin-top: 8px;`;
```

Comment:
Something minor and what you couldn't have known, but I like to keep spacing on a scale of factors of two where possible, so `4px`, `8px`, `16px`, etc. There are exceptions to this, but other buttons with icons on the site have a gap of `8px` between the icon and the text, so we should probably keep the consistency here.

Regarding the `margin-top`: The parent element already applies a `gap` between elements, so I think we can get rid of the margin here, otherwise it will stack which results in an off-scale spacing again.